pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: recco
 description: Recco is a personalized, and user-friendly approach to health and wellness recommendations based on users' unique interests, motivations, and health behaviors.
-version: 0.1.3
+version: 1.0.0
```

Comment:
I guess recco.podspec should be in line with the 1.0.0 version

Reply:
@arubio will do that change